things/service_test.go

```diff
@@ -24,11 +24,14 @@ const (
 	token      = "token"
 	token2     = "token2"
 	n          = uint64(10)
+	Prefix     = "fe6b4e92-cc98-425e-b0aa-"
```

Comment:
Same here, why this is exported? Maybe we can use `uuidPrefix`

Reply:
same as above.